tailnet/test/integration/integration.go

```diff
@@ -171,6 +203,21 @@ func (o SimpleServerOptions) Router(t *testing.T, logger slog.Logger) *chi.Mux {
 		r.Get("/latency-check", func(w http.ResponseWriter, r *http.Request) {
 			w.WriteHeader(http.StatusOK)
 		})
+		r.Post("/restart", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Maybe this could be refactored into a single endpoint such as `/restart` (at the root of the router) which takes query params for what you want to restart. `/restart?derp=true&coordinator=true` would simulate a full coderd restart for example

Having the two endpoints seems kinda weird, it would also make the `sendRestart` function feel less out of place (it's odd that it takes a *path*)